package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "cypress-tests",
+  "author": "CivicActions",
+  "version": "0.1.0",
+  "description": "A repository to store Cypress test recipes created by CivicActions engineers",
+  "main": "index.js",
+  "repository": "git@github.com:CivicActions/cypress-tests.git",
+  "license": "MIT",
+  "private": false,
+  "dependencies": {
+    "cypress": "^11.2.0"
```

Comment:
Looks like cypress-axe just merged support for version 11 (like an hour ago https://github.com/component-driven/cypress-axe/pull/143) so there might be a release coming out soon. However, if we install cypress-axe now it would push cypress version to 10.

Reply:
Hmm, I've been using Cypress 11.x with cypress-axe "0.13.0" and there didn't seem to be issues, but I think once GitHub Actions are set up on this repo with both dependencies, we should know when breaking changes arise. 

So, I'm fine merging this in now and fixing whatever needs fixed later. In fact, we could even help fix compatibility issues in those project if we try to keep all dependencies as up-to-date as possible. 

Thoughts @dmundra ?

Reply:
Sounds good to me.